core/src/main/scala/org/ensime/config/EnsimeConfigProtocol.scala

```diff
@@ -28,17 +27,18 @@ object EnsimeConfigProtocol {
   private implicit val moduleFormat: SexpFormat[EnsimeModule] = cachedImplicit
   private implicit val configFormat: SexpFormat[EnsimeConfig] = cachedImplicit
 
-  def parse(config: String): EnsimeConfig = {
+  def parse(config: String, serverConfig: String): EnsimeConfig = {
```

Comment:
I think we want a HOCON file though, I'm undecided. The s-expression format is difficult for a lot of editors to handle... so I'm even thinking that `.ensime` should be `.ensime.conf` in HOCON format.